dDatabase/CDClientDatabase/CDClientTables/CDMissionEmailTable.cpp

```diff
@@ -25,7 +25,7 @@ void CDMissionEmailTable::LoadValuesFromDatabase() {
 		entry.notificationGroup = tableData.getIntField("notificationGroup", -1);
 		entry.missionID = tableData.getIntField("missionID", -1);
 		entry.attachmentLOT = tableData.getIntField("attachmentLOT", 0);
-		entry.localize = (bool)tableData.getIntField("localize", -1);
+		entry.localize = static_cast<bool>(tableData.getIntField("localize", -1));
```

Comment:
hm, an intended bool defaulting to -1 is kind of odd, this should probably just default to 0 instead.  Thoughts?

Reply:
default should be true (so 1)

Reply:
changed